### PR TITLE
Issue 18542 field sort order

### PR DESCRIPTION
**Pull request by wezell**

### CODE REVIEW

File: dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java [hunk 1]
```diff
@@ -23,22 +21,22 @@ public static FieldSql getInstance() {
...
 	
-	public String findByContentTypeAndFieldVar = SELECT_ALL_FIELDS + " where inode.inode = field.inode and structure_inode =? and field.velocity_var_name=?";
+	public final String findByContentTypeAndFieldVar = SELECT_ALL_FIELDS + " where inode.inode = field.inode and structure_inode =? and field.velocity_var_name=?";
 	
 	
-	public String deleteByContentType = "delete from field where structure_inode = ?";
-	public String deleteById = "delete from field where inode = ?";
-	public String deleteInodeById = "delete from inode where inode = ? and type='field'";
+	public final String deleteByContentType = "delete from field where structure_inode = ?";
+	public final String deleteById = "delete from field where inode = ?";
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java:36]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5572908)

File: dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java [hunk 2]
```diff
@@ -90,23 +88,23 @@ public static FieldSql getInstance() {
...
-	public String selectFieldVarByKey= 	"select id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date from field_variable where variable_key = ? ";
-	public String selectFieldIdVarByKey= 	"select id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date from field_variable where field_id = ? and variable_key = ?";
+	public final String selectFieldVars= 	"select id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date from field_variable where field_id = ? order by variable_key";
+	public final String selectFieldVar= 	"select id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date from field_variable where id = ? ";
+	public final String selectFieldVarByKey= 	"select id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date from field_variable where variable_key = ? ";
+	public final String selectFieldIdVarByKey= 	"select id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date from field_variable where field_id = ? and variable_key = ?";
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java:103]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5572908)

File: dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java [hunk 3]
```diff
@@ -60,11 +58,11 @@ public static FieldSql getInstance() {
 			+ "mod_date=? "
 			+ "where inode =?";
 	
-	public String updateFieldInode = "update inode set inode=?, idate=?, owner = ? where inode = ? and type='field'";
+	public final String updateFieldInode = "update inode set inode=?, idate=?, owner = ? where inode = ? and type='field'";
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java:61]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5572908)

File: dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java [hunk 4]
```diff
@@ -90,23 +88,23 @@ public static FieldSql getInstance() {
...
+	public final String selectFieldVarByKey= 	"select id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date from field_variable where variable_key = ? ";
+	public final String selectFieldIdVarByKey= 	"select id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date from field_variable where field_id = ? and variable_key = ?";
 
-    public String deleteFieldVar= 	"delete from field_variable where id = ? or (field_id=? and variable_key=?)";
-	public String deleteFieldVarsForField= "delete from field_variable where field_id = ?";
-	public String insertFieldVar= 	"insert into field_variable( id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date) values (?,?,?,?,?,?,?)";
+    public final String deleteFieldVar= 	"delete from field_variable where id = ? or (field_id=? and variable_key=?)";
+	public final String deleteFieldVarsForField= "delete from field_variable where field_id = ?";
+	public final String insertFieldVar= 	"insert into field_variable( id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date) values (?,?,?,?,?,?,?)";
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java:107]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5572908)

File: dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java [hunk 5]
```diff
@@ -90,23 +88,23 @@ public static FieldSql getInstance() {
...
 	
-	public String selectFieldVars= 	"select id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date from field_variable where field_id = ? order by variable_key";
-	public String selectFieldVar= 	"select id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date from field_variable where id = ? ";
-	public String selectFieldVarByKey= 	"select id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date from field_variable where variable_key = ? ";
-	public String selectFieldIdVarByKey= 	"select id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date from field_variable where field_id = ? and variable_key = ?";
+	public final String selectFieldVars= 	"select id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date from field_variable where field_id = ? order by variable_key";
+	public final String selectFieldVar= 	"select id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date from field_variable where id = ? ";
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java:101]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5572908)

File: dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java [hunk 6]
```diff
@@ -60,11 +58,11 @@ public static FieldSql getInstance() {
 			+ "mod_date=? "
 			+ "where inode =?";
 	
-	public String updateFieldInode = "update inode set inode=?, idate=?, owner = ? where inode = ? and type='field'";
+	public final String updateFieldInode = "update inode set inode=?, idate=?, owner = ? where inode = ? and type='field'";
 	
-	public String insertFieldInode = "insert into inode (inode, idate, owner, type) values (?,?,?,'field')";
+	public final String insertFieldInode = "insert into inode (inode, idate, owner, type) values (?,?,?,'field')";
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java:63]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5572908)

File: dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java [hunk 7]
```diff
@@ -90,23 +88,23 @@ public static FieldSql getInstance() {
...
-	public String selectCountOfType= "select count(*) as test from field where structure_inode = ? and field_type like ? or field_type like ?";
+	public final String selectCountOfType= "select count(*) as test from field where structure_inode = ? and field_type like ? or field_type like ?";
 	
-	public String selectFieldVars= 	"select id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date from field_variable where field_id = ? order by variable_key";
-	public String selectFieldVar= 	"select id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date from field_variable where id = ? ";
-	public String selectFieldVarByKey= 	"select id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date from field_variable where variable_key = ? ";
-	public String selectFieldIdVarByKey= 	"select id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date from field_variable where field_id = ? and variable_key = ?";
+	public final String selectFieldVars= 	"select id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date from field_variable where field_id = ? order by variable_key";
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java:100]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5572908)

File: dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java [hunk 8]
```diff
@@ -90,23 +88,23 @@ public static FieldSql getInstance() {
 	
 	
 	
-	public String inodeCount = "select count(inode) as inode_count from field where inode = ?";
+	public final String inodeCount = "select count(inode) as inode_count from field where inode = ?";
 	
 	
 	
-	public String selectFieldOfDbType = "select field_contentlet from field where structure_inode = ? and field_contentlet like ? order by field_contentlet";
+	public final String selectFieldOfDbType = "select field_contentlet from field where structure_inode = ? and field_contentlet like ? order by field_contentlet";
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java:95]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5572908)

File: dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java [hunk 9]
```diff
@@ -90,23 +88,23 @@ public static FieldSql getInstance() {
...
+	public final String selectFieldVar= 	"select id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date from field_variable where id = ? ";
+	public final String selectFieldVarByKey= 	"select id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date from field_variable where variable_key = ? ";
+	public final String selectFieldIdVarByKey= 	"select id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date from field_variable where field_id = ? and variable_key = ?";
 
-    public String deleteFieldVar= 	"delete from field_variable where id = ? or (field_id=? and variable_key=?)";
-	public String deleteFieldVarsForField= "delete from field_variable where field_id = ?";
-	public String insertFieldVar= 	"insert into field_variable( id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date) values (?,?,?,?,?,?,?)";
+    public final String deleteFieldVar= 	"delete from field_variable where id = ? or (field_id=? and variable_key=?)";
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java:105]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5572908)

File: dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java [hunk 10]
```diff
@@ -90,23 +88,23 @@ public static FieldSql getInstance() {
...
-	public String selectFieldVar= 	"select id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date from field_variable where id = ? ";
-	public String selectFieldVarByKey= 	"select id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date from field_variable where variable_key = ? ";
-	public String selectFieldIdVarByKey= 	"select id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date from field_variable where field_id = ? and variable_key = ?";
+	public final String selectFieldVars= 	"select id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date from field_variable where field_id = ? order by variable_key";
+	public final String selectFieldVar= 	"select id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date from field_variable where id = ? ";
+	public final String selectFieldVarByKey= 	"select id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date from field_variable where variable_key = ? ";
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java:102]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5572908)

File: dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java [hunk 11]
```diff
@@ -23,22 +21,22 @@ public static FieldSql getInstance() {
...
 	
 	
-	public String findByContentTypeAndFieldVar = SELECT_ALL_FIELDS + " where inode.inode = field.inode and structure_inode =? and field.velocity_var_name=?";
+	public final String findByContentTypeAndFieldVar = SELECT_ALL_FIELDS + " where inode.inode = field.inode and structure_inode =? and field.velocity_var_name=?";
 	
 	
-	public String deleteByContentType = "delete from field where structure_inode = ?";
-	public String deleteById = "delete from field where inode = ?";
-	public String deleteInodeById = "delete from inode where inode = ? and type='field'";
+	public final String deleteByContentType = "delete from field where structure_inode = ?";
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java:35]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5572908)

File: dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java [hunk 12]
```diff
@@ -90,23 +88,23 @@ public static FieldSql getInstance() {
 	
 	
 	
-	public String inodeCount = "select count(inode) as inode_count from field where inode = ?";
+	public final String inodeCount = "select count(inode) as inode_count from field where inode = ?";
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java:91]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5572908)

File: dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java [hunk 13]
```diff
@@ -90,23 +88,23 @@ public static FieldSql getInstance() {
...
+	public final String selectFieldVar= 	"select id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date from field_variable where id = ? ";
+	public final String selectFieldVarByKey= 	"select id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date from field_variable where variable_key = ? ";
+	public final String selectFieldIdVarByKey= 	"select id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date from field_variable where field_id = ? and variable_key = ?";
 
-    public String deleteFieldVar= 	"delete from field_variable where id = ? or (field_id=? and variable_key=?)";
-	public String deleteFieldVarsForField= "delete from field_variable where field_id = ?";
-	public String insertFieldVar= 	"insert into field_variable( id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date) values (?,?,?,?,?,?,?)";
+    public final String deleteFieldVar= 	"delete from field_variable where id = ? or (field_id=? and variable_key=?)";
+	public final String deleteFieldVarsForField= "delete from field_variable where field_id = ?";
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java:106]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5572908)

File: dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java [hunk 14]
```diff
@@ -90,23 +88,23 @@ public static FieldSql getInstance() {
...
+	public final String inodeCount = "select count(inode) as inode_count from field where inode = ?";
 	
 	
 	
-	public String selectFieldOfDbType = "select field_contentlet from field where structure_inode = ? and field_contentlet like ? order by field_contentlet";
+	public final String selectFieldOfDbType = "select field_contentlet from field where structure_inode = ? and field_contentlet like ? order by field_contentlet";
 	
 	
-	public String selectCountOfType= "select count(*) as test from field where structure_inode = ? and field_type like ? or field_type like ?";
+	public final String selectCountOfType= "select count(*) as test from field where structure_inode = ? and field_type like ? or field_type like ?";
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java:98]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5572908)

File: dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java [hunk 15]
```diff
@@ -23,22 +21,22 @@ public static FieldSql getInstance() {
...
-	public String findByContentTypeAndFieldVar = SELECT_ALL_FIELDS + " where inode.inode = field.inode and structure_inode =? and field.velocity_var_name=?";
+	public final String findByContentTypeAndFieldVar = SELECT_ALL_FIELDS + " where inode.inode = field.inode and structure_inode =? and field.velocity_var_name=?";
 	
 	
-	public String deleteByContentType = "delete from field where structure_inode = ?";
-	public String deleteById = "delete from field where inode = ?";
-	public String deleteInodeById = "delete from inode where inode = ? and type='field'";
+	public final String deleteByContentType = "delete from field where structure_inode = ?";
+	public final String deleteById = "delete from field where inode = ?";
+	public final String deleteInodeById = "delete from inode where inode = ? and type='field'";
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java:37]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5572908)

**Review by nollymar — APPROVED** [review]

**Review by freddyucv — APPROVED** [review]

**Review by erickgonzalez — APPROVED** [review]

File: dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java (hunk 3 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java:61]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5572908)

File: dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java (hunk 15 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java:37]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5572908)

File: dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java (hunk 12 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java:91]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5572908)

File: dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java (hunk 5 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java:101]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5572908)

File: dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java (hunk 4 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java:107]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5572908)

File: dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java (hunk 10 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java:102]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5572908)

File: dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java (hunk 8 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java:95]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5572908)

File: dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java (hunk 1 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java:36]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5572908)

File: dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java (hunk 13 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java:106]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5572908)

File: dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java (hunk 7 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java:100]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5572908)

File: dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java (hunk 2 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java:103]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5572908)

File: dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java (hunk 6 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java:63]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5572908)

File: dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java (hunk 11 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java:35]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5572908)

File: dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java (hunk 14 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java:98]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5572908)

File: dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java (hunk 9 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java:105]: Codacy Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5572908)

**Pull request merged**